cluster_test.go

```diff
@@ -61,31 +61,28 @@ var _ = Describe("Tests", func() {
 		Context("PurgeCache", func() {
 			It("should remove all contents from memory", func() {
 				time.Sleep(5 * time.Second)
-				stats, err := handle.Statistics()
+				stats, err := secureHandle.Statistics()
 				Expect(err).ToNot(HaveOccurred())
-				var nodeStat Statistics
+				Expect(len(stats)).To(BeNumerically(">", 0))
+
+				var memBefore int64
 				for _, stat := range stats {
-					nodeStat = stat
+					memBefore = stat.Memory.BytesResident
 					break
 				}
```

Comment:
Why do you use for loop here and not access the elem directly?

Reply:
I was thinking something like
`
if len(stats) > 0 {
    memBefore = stats[0].Memory.BytesResident
}
`